<?xml version="1.0" encoding="utf-8" ?>
<dwsync>
<file name="__init__.py" server="192.168.0.12//root/alligator/hydrogen/" local="130472753023977564" remote="130473522830000000" Dst="0" />
<file name="svpxyexception.py" server="192.168.0.12//root/alligator/hydrogen/" local="130472758050563079" remote="130473522830000000" Dst="0" />
<file name="__init__.py" server="192.168.0.13//root/alligatorproxy/" local="130472753023970000" remote="130473545200000000" Dst="0" />
<file name="svpxyexception.py" server="192.168.0.13//root/alligatorproxy/" local="130472758050560000" remote="130473545210000000" Dst="0" />
<file name="__init__.py" server="192.168.0.13//root/alligatorproxy/hydrogen/" local="130472753023970000" remote="130473547100000000" Dst="0" />
<file name="svpxyexception.py" server="192.168.0.13//root/alligatorproxy/hydrogen/" local="130473557812463154" remote="130473557860000000" Dst="0" />
<file name="svpxyexception.py" server="192.168.0.14//root/alligatorproxy/hydrogen/" local="130516775069209082" remote="130516815030000000" Dst="0" />
<file name="__init__.py" server="192.168.0.14//root/alligatorproxy/hydrogen/" local="130516759988040056" remote="130516815030000000" Dst="0" />
</dwsync>